Makefile

```diff
@@ -1,11 +1,12 @@
+TARGET        = kata-ksm-throttler
```

Comment:
The whole PR looks fine, but I'd prefer if we do not make this kata specific. I think this could be used in non kata contexts and although at the moment there's only one vc based trigger, other non kata triggers could be added.
I also want to add a non trigger mode, where ksm-throttler only looks at meminfo.

Reply:
Ok. With respect to https://github.com/clearcontainers/packaging/pull/200, how about I change the default target back to `ksm-throttler` so that we can build it as either `cc-ksm-throttler` or `kata-ksm-throttler` as desired?

Reply:
Yes, that is fine with me.